.travis.yml

```diff
@@ -123,6 +123,7 @@ before_install:
         apt-key adv --list-public-keys --with-colons | grep '^pub' | cut -d':' -f 5 | egrep -o '.{8}$' | grep -wo 762E3157 > /dev/null 2>&1 || sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 762E3157;
       fi
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+        pip3 install travis-wait-improved;
```

Comment:
@stefanv @jni @emmanuelle 

This is exactly the kind of frustration that our users experience when they try to install our software.

They attempt to use:


```
pip install scikit-image
```
but then there is always some edge case.

Is this something we want to document?

If so, is a "troubleshooting" section appropriate?

Reply:
A troubleshooting section could be a useful addition indeed.